flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/mailbox/TaskMailboxImpl.java

```diff
@@ -27,19 +27,19 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Iterator;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.locks.Condition;
 import java.util.concurrent.locks.ReentrantLock;
 
 /**
- * Implementation of {@link Mailbox} inspired by {@link java.util.concurrent.ArrayBlockingQueue} and tailored towards
+ * Implementation of {@link TaskMailbox} inspired by {@link java.util.concurrent.ArrayBlockingQueue} and tailored towards
```

Comment:
Not related to changes in this PR, but `ArrayBlockingQueue` notice is outdated. The implementation is not using ring buffer anymore.

Reply:
I changed the comment to a more general reference to blocking queue.